### PR TITLE
Release Notes for iOS Release 1.63.183

### PR DESCRIPTION
**Pull request by Uni-verse**
Resolves https://github.com/brave/brave-browser/issues/36937

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A
  - [ ] https://github.com/brave/brave-browser/wiki/Brave-Wallet

## Test Plan:
N/A

### CODE REVIEW

File: CHANGELOG_iOS.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+### General
+
+- Added Brave Leo - AI for questions and answers leveraging page context. ([#36585](https://github.com/brave/brave-browser/issues/36585))
+- Added “Copy Clean Link” to the share menu. ([#8070](https://github.com/brave/brave-ios/issues/8070))
+- Added interstitial pages when blocking pages (top-level domain blocking). ([#8096](https://github.com/brave/brave-ios/issues/8096))
+- Added “Open in Brave” to iOS Share Sheet menu. ([#8516](https://github.com/brave/brave-ios/issues/8516))
+- Added opening scanned URLs in a new tab when selecting “Scan QR Code” from the Home Screen context menu. ([#8594](https://github.com/brave/brave-ios/issues/8594))
+- Removed the limit on the number of filter lists that can be selected at once in “Content Filtering” page. ([#8656](https://github.com/brave/brave-ios/issues/8656))
+- Updated Webcompat report to use the site URL instead of using “internal://local”. ([#8667](https://github.com/brave/brave-ios/issues/8667))
```
Comment (kjozwiak) [inline CHANGELOG_iOS.md:18]: nit: Don't need the quotes around `internal://local`. We usually don't put quotes around `brave://`, `chrome://` or `internal://` mentions.

File: CHANGELOG_iOS.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- Fixed transaction status not updating automatically in "Transaction Details" page. ([#36468](https://github.com/brave/brave-browser/issues/36468))
+
+### General
+
+- Added Brave Leo - AI for questions and answers leveraging page context. ([#36585](https://github.com/brave/brave-browser/issues/36585))
+- Added “Copy Clean Link” to the share menu. ([#8070](https://github.com/brave/brave-ios/issues/8070))
+- Added interstitial pages when blocking pages (top-level domain blocking). ([#8096](https://github.com/brave/brave-ios/issues/8096))
+- Added “Open in Brave” to iOS Share Sheet menu. ([#8516](https://github.com/brave/brave-ios/issues/8516))
+- Added opening scanned URLs in a new tab when selecting “Scan QR Code” from the Home Screen context menu. ([#8594](https://github.com/brave/brave-ios/issues/8594))
+- Removed the limit on the number of filter lists that can be selected at once in “Content Filtering” page. ([#8656](https://github.com/brave/brave-ios/issues/8656))
```
Comment (kjozwiak) [inline CHANGELOG_iOS.md:17]: hmm I think `under` mind sound better in this case. For example, something similar to:

```
- Removed the limit on the number of filter lists that can be selected at once under “Content Filtering” settings page. ([#8656](https://github.com/brave/brave-ios/issues/8656))
```

File: CHANGELOG_iOS.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+- Implemented v2 UI for “Accounts” tab in Brave Wallet. ([#8600](https://github.com/brave/brave-ios/issues/8600))
+- Fixed transaction status not updating automatically in "Transaction Details" page. ([#36468](https://github.com/brave/brave-browser/issues/36468))
+
+### General
+
+- Added Brave Leo - AI for questions and answers leveraging page context. ([#36585](https://github.com/brave/brave-browser/issues/36585))
+- Added “Copy Clean Link” to the share menu. ([#8070](https://github.com/brave/brave-ios/issues/8070))
+- Added interstitial pages when blocking pages (top-level domain blocking). ([#8096](https://github.com/brave/brave-ios/issues/8096))
+- Added “Open in Brave” to iOS Share Sheet menu. ([#8516](https://github.com/brave/brave-ios/issues/8516))
+- Added opening scanned URLs in a new tab when selecting “Scan QR Code” from the Home Screen context menu. ([#8594](https://github.com/brave/brave-ios/issues/8594))
```
Comment (kjozwiak) [inline CHANGELOG_iOS.md:16]: How about something like this? I don't think we `Added` the ability, just updated the behaviour. But I could be wrong.

```
 - Updated “Scan QR Code” to open scanned URLs in a new tab from the "Home Screen" context menu. ([#8594](https://github.com/brave/brave-ios/issues/8594))
```

File: CHANGELOG_iOS.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...

+- Implemented v2 UI for "Account" details in Brave Wallet. ([#8663](https://github.com/brave/brave-ios/issues/8663))
+- Implemented v2 UI for “Accounts” tab in Brave Wallet. ([#8600](https://github.com/brave/brave-ios/issues/8600))
+- Fixed transaction status not updating automatically in "Transaction Details" page. ([#36468](https://github.com/brave/brave-browser/issues/36468))
+
+### General
+
+- Added Brave Leo - AI for questions and answers leveraging page context. ([#36585](https://github.com/brave/brave-browser/issues/36585))
+- Added “Copy Clean Link” to the share menu. ([#8070](https://github.com/brave/brave-ios/issues/8070))
+- Added interstitial pages when blocking pages (top-level domain blocking). ([#8096](https://github.com/brave/brave-ios/issues/8096))
```
Comment (kjozwiak) [inline CHANGELOG_iOS.md:14]: How about something a bit cleaner like the following?

```
- Added interstitial pages when blocking top-level domain pages. ([#8096](https://github.com/brave/brave-ios/issues/8096))
```

**Review by kjozwiak — CHANGES_REQUESTED** [review]
You'll need to update the spacing of each point so its matching `Desktop` & `Android`. There's also a missing space between the title of `Web3` and the list. It should look something as the following:

```
## [1.63.183](https://github.com/brave/brave-browser/releases/tag/v1.63.183)

### Web3

 - Implemented v2 UI for "Account" details in Brave Wallet. ([#8663](https://github.com/brave/brave-ios/issues/8663))
 - Implemented v2 UI for “Accounts” tab in Brave Wallet. ([#8600](https://github.com/brave/brave-ios/issues/8600))
 - Fixed transaction status not updating automatically in "Transaction Details" page. ([#36468](https://github.com/brave/brave-browser/issues/36468))

### General

 - Added Brave Leo - AI for questions and answers leveraging page context. ([#36585](https://github.com/brave/brave-browser/issues/36585))
 - Added “Copy Clean Link” to the share menu. ([#8070](https://github.com/brave/brave-ios/issues/8070))
 - Added interstitial pages when blocking pages (top-level domain blocking). ([#8096](https://github.com/brave/brave-ios/issues/8096))
 - Added “Open in Brave” to iOS Share Sheet menu. ([#8516](https://github.com/brave/brave-ios/issues/8516))
 - Added opening scanned URLs in a new tab when selecting “Scan QR Code” from the Home Screen context menu. ([#8594](https://github.com/brave/brave-ios/issues/8594))
 - Removed the limit on the number of filter lists that can be selected at once in “Content Filtering” page. ([#8656](https://github.com/brave/brave-ios/issues/8656))
 - Updated Webcompat report to use the site URL instead of using “internal://local”. ([#8667](https://github.com/brave/brave-ios/issues/8667))
 - Fixed an issue with muting/unmuting when Brave Talk is minimized. ([#8717](https://github.com/brave/brave-ios/issues/8717))
 - Fixed an issue with the search suggestions button not updating when pressed in the search field. ([#8630](https://github.com/brave/brave-ios/issues/8630))
 ```
 
 Easiest way to check is make sure you're `iOS` notes align re: spacing with both https://raw.githubusercontent.com/brave/brave-browser/master/CHANGELOG_DESKTOP.md & https://raw.githubusercontent.com/brave/brave-browser/master/CHANGELOG_ANDROID.md.

File: CHANGELOG_iOS.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+## [1.63.183](https://github.com/brave/brave-browser/releases/tag/v1.63.183)
+
+### Web3
```
Comment (kjozwiak) [inline CHANGELOG_iOS.md:5]: nit: tab/white space in front of `Web3`

image

**Review by kjozwiak — APPROVED** [review]
LGTM other than just removing those two spaces after `Web3`.

File: CHANGELOG_iOS.md (hunk 5 above)
Comment (Uni-verse) [reply CHANGELOG_iOS.md]: Got it, nice catch.

**Pull request merged**